### PR TITLE
Feat/#13 devices component

### PR DESCRIPTION
**Pull request by Jaimeandres97**
## Description

Creation of device component in index.html and their styling for mobile, tablet and desktop

Feat/#13 devices component

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [X] Vite

**Test Configuration**:

- Dependencies versions:
- Node version

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

**Comment by Jaimeandres97** [comment]
Because I used the refactor branch to start this feature I bring the previous commits and didn't realize about that so once it was uploaded I didn't know how to squash the commits (again). And again... so sorry :(

File: index.html [hunk 1]
```diff
@@ -100,6 +100,41 @@ <h3 class="plan__price">COP 319.900/año*</h3>
...
+    <section class="devices">
+      <h2 class="devices__title">Disponible en tus dispositivos favoritos</h2>
+      <ul class="devices-container">
+        <li class="devices-container__content">
+          <img class="devices__img" src="./img/pc.png" />
+          <h3 class="devices__options">Computadoras</h3>
+          <p class="devices__description">
+            Chrome OS<br />MacOS<br />PC Windows
+          </p>
+        </li>
```
Comment (juancho11gm) [inline index.html]: Let's use `figure` and `figcaption`. Please read the [docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/figure).

Please tell me what do you think about this refactor.
```suggestion
<li class="devices-container__content">
          <figure class="device">
            <img class="device__img" src="./img/pc.png" />
            <h3 class="device__options">Computadoras</h3>
            <figcaption class="device__description">
              <ul class="devices__items">
                <li class="devices__item">Chrome OSM</li>
                <li class="devices__item">MacOS</li>
                <li class="devices__item">PC Windows</li>
              </ul>
            </figcaption>
          </figure>
        </li>
```

File: src/components/Devices/Devices.scss [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+  font-size: 15px;
+  line-height: 25px;
+  margin: 0 0 24px;
+  margin-bottom: 20px;
+
+  @include desktop {
+    font-size: 18px;
+    line-height: 28px;
+  }
+}
```
Comment (juancho11gm) [inline src/components/Devices/Devices.scss:78]:
```suggestion
@import '../../styles/mixins';
@import '../../styles/main.scss';

.devices {
  padding: 10vw 5vw;
  text-align: center;

  &__title {
    color: var(--text-primary-color);
    font-size: 26px;
    letter-spacing: normal;
    line-height: 36px;
    margin-bottom: 16px;
    margin-top: 0;

    @include desktop {
      font-size: 40px;
      line-height: 50px;
      margin-bottom: 24px;
    }
  }

  .devices-container {
    display: flex;
    flex-wrap: wrap;
    justify-content: space-around;
    margin: 0 -12px;
    padding-left: 0;

    &__content {
      box-sizing: border-box;
      display: flex;
      flex: 0 0 50%;
      flex-direction: column;
      list-style: none;
      max-width: 50%;
      padding: 12px;

      @include desktop {
        flex: 0 0 25%;
        max-width: 25%;
      }
    }

    &__img {
      width: 100%;
      height: auto;
      display: block;
    }

    &__options {
      font-size: 22px;
      letter-spacing: normal;
      line-height: 32px;
      margin-bottom: 16px;
      margin-top: 20px;

      @include desktop {
        font-size: 28px;
        line-height: 38px;
        margin-bottom: 24px;
      }
    }

    &__description {
      color: var(--secondary-color);
      font-size: 15px;
      line-height: 25px;
      margin: 0 0 24px;
      margin-bottom: 20px;

      @include desktop {
        font-size: 18px;
        line-height: 28px;
      }
    }
  }
}
```

Note: the classnames might change according to the refactor suggested in the HTML.

Comment (juancho11gm) [inline src/components/Devices/Devices.scss:78]:
```suggestion
@import '../../styles/mixins';
@import '../../styles/main.scss';

.devices {
  padding: 10vw 5vw;
  text-align: center;

  &__title {
    color: var(--text-primary-color);
    font-size: 26px;
    letter-spacing: normal;
    line-height: 36px;
    margin-bottom: 16px;
    margin-top: 0;

    @include desktop {
      font-size: 40px;
      line-height: 50px;
      margin-bottom: 24px;
    }
  }

  .devices-container {
    display: flex;
    flex-wrap: wrap;
    justify-content: space-around;
    margin: 0 -12px;
    padding-left: 0;

    &__content {
      box-sizing: border-box;
      display: flex;
      flex: 0 0 50%;
      flex-direction: column;
      list-style: none;
      max-width: 50%;
      padding: 12px;

      @include desktop {
        flex: 0 0 25%;
        max-width: 25%;
      }
    }

    &__img {
      width: 100%;
      height: auto;
      display: block;
    }

    &__options {
      font-size: 22px;
      letter-spacing: normal;
      line-height: 32px;
      margin-bottom: 16px;
      margin-top: 20px;

      @include desktop {
        font-size: 28px;
        line-height: 38px;
        margin-bottom: 24px;
      }
    }

    &__description {
      color: var(--secondary-color);
      font-size: 15px;
      line-height: 25px;
      margin: 0 0 24px;
      margin-bottom: 20px;

      @include desktop {
        font-size: 18px;
        line-height: 28px;
      }
    }
  }
}
```

Note: the classnames might change according to the refactor suggested in the HTML.

Comment (juancho11gm) [inline src/components/Devices/Devices.scss]:
```suggestion
@import '../../styles/mixins';
@import '../../styles/main.scss';

.devices {
  padding: 10vw 5vw;
  text-align: center;

  &__title {
    color: var(--text-primary-color);
    font-size: 26px;
    letter-spacing: normal;
    line-height: 36px;
    margin-bottom: 16px;
    margin-top: 0;

    @include desktop {
      font-size: 40px;
      line-height: 50px;
      margin-bottom: 24px;
    }
  }

  .devices-container {
    display: flex;
    flex-wrap: wrap;
    justify-content: space-around;
    margin: 0 -12px;
    padding-left: 0;

    &__content {
      box-sizing: border-box;
      display: flex;
      flex: 0 0 50%;
      flex-direction: column;
      list-style: none;
      max-width: 50%;
      padding: 12px;

      @include desktop {
        flex: 0 0 25%;
        max-width: 25%;
      }
    }

    &__img {
      width: 100%;
      height: auto;
      display: block;
    }

    &__options {
      font-size: 22px;
      letter-spacing: normal;
      line-height: 32px;
      margin-bottom: 16px;
      margin-top: 20px;

      @include desktop {
        font-size: 28px;
        line-height: 38px;
        margin-bottom: 24px;
      }
    }

    &__description {
      color: var(--secondary-color);
      font-size: 15px;
      line-height: 25px;
      margin: 0 0 24px;
      margin-bottom: 20px;

      @include desktop {
        font-size: 18px;
        line-height: 28px;
      }
    }
  }
}
```

Note: the classnames might change according to the refactor suggested in the HTML.

File: src/components/Hero/Hero.scss [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+  @include desktop {
+    height: 100%;
+    font-size: 18px;
+    line-height: 18px;
+    max-width: 700px;
+  }
+}
+
+.hero__link {
+  color: var(--text-primary-color);
```
Comment (juancho11gm) [inline src/components/Hero/Hero.scss]: please use [scss variables](https://sass-lang.com/documentation/variables).

File: src/components/Devices/Devices.scss (hunk 2 above)
Comment (juancho11gm) [inline src/components/Devices/Devices.scss:78]:
```scss
@import '../../styles/mixins';
@import '../../styles/main.scss';

.devices {
  padding: 10vw 5vw;
  text-align: center;

  &__title {
    color: var(--text-primary-color);
    font-size: 26px;
    letter-spacing: normal;
    line-height: 36px;
    margin-bottom: 16px;
    margin-top: 0;

    @include desktop {
      font-size: 40px;
      line-height: 50px;
      margin-bottom: 24px;
    }
  }

  .devices-container {
    display: flex;
    flex-wrap: wrap;
    justify-content: space-around;
    margin: 0 -12px;
    padding-left: 0;

    &__content {
      box-sizing: border-box;
      display: flex;
      flex: 0 0 50%;
      flex-direction: column;
      list-style: none;
      max-width: 50%;
      padding: 12px;

      @include desktop {
        flex: 0 0 25%;
        max-width: 25%;
      }
    }

    &__img {
      width: 100%;
      height: auto;
      display: block;
    }

    &__options {
      font-size: 22px;
      letter-spacing: normal;
      line-height: 32px;
      margin-bottom: 16px;
      margin-top: 20px;

      @include desktop {
        font-size: 28px;
        line-height: 38px;
        margin-bottom: 24px;
      }
    }

    &__description {
      color: var(--secondary-color);
      font-size: 15px;
      line-height: 25px;
      margin: 0 0 24px;
      margin-bottom: 20px;

      @include desktop {
        font-size: 18px;
        line-height: 28px;
      }
    }
  }
}
```

Note: the classnames might change according to the refactor suggested in the HTML.

**Review by juancho11gm — COMMENTED** [review]
- Please save the images under the **img/devices** folder.
- Please remember to upload an image of your feature.
Nice job.

File: index.html (hunk 1 above)
Comment (Jaimeandres97) [reply index.html]: I think this change will improve in the responsive design of the webpage. Please tell me if that is correct and if I am missing something about the reason of this refactor

**Comment by Jaimeandres97** [comment]
MOBILE DEVICES
PC DEVICES

File: index.html (hunk 1 above)
Comment (juancho11gm) [reply index.html]: It’s just about semantic and accessibility

File: src/components/Plans/Plans.scss [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+  }
+}
+
+.plan--monthly {
+  background-image: url('../img/plans/month-year.png');
+}
+
+.plan--annually {
+  background-image: url('../img/plans/month-year.png');
+}
```
Comment (juancho11gm) [inline src/components/Plans/Plans.scss]: you can include all these selectors within the ` .plan` code block

Comment (juancho11gm) [reply src/components/Plans/Plans.scss]: nice job :)

**Review by juancho11gm — APPROVED** [review]

**Pull request merged**